locations/spiders/montre_service_fr.py

```
@@ -0,0 +1,36 @@
+    rules = [
+        Rule(
+            LinkExtractor(allow=r"/boutiques/.*"),
+            "parse_sd",
+            follow=True,
+        ),
+    ]
+
+    drop_attributes = {"facebook"}
+
+    def post_process_item(self, item, response, ld_data):
+        if(item.get("street_address") == None): #shops for which the structured data spider cannot extract street_address are invalid
```

Flake8 (7.3.0)

[error] comparison to None should be 'if cond is None:'

(E711)